ckks/params.go

```diff
@@ -14,6 +14,9 @@ import (
 // MaxLogN is the log2 of the largest supported polynomial modulus degree.
 const MaxLogN = 16
 
+// MinLogN is the log2 of the smallest supported polynomial modulus degree
+const MinLogN = 4
```

Comment:
Same as for BFV comment

Reply:
Minimum must be 2^4 to ensure the NTT correctness.

Reply:
Ok for me. Mention this in the comment too.

Reply:
Done